lib/curses.neon

```diff
@@ -12,6 +12,262 @@ DECLARE NATIVE CONST OK: Number
+DECLARE NATIVE CONST KEY_MESSAGE        :Number % message key
+DECLARE NATIVE CONST KEY_MOVE           :Number % move key
+DECLARE NATIVE CONST KEY_NEXT           :Number % next object key
+DECLARE NATIVE CONST KEY_OPEN           :Number % open key
+DECLARE NATIVE CONST KEY_OPTIONS        :Number % options key
+DECLARE NATIVE CONST KEY_PREVIOUS       :Number % previous object key
+DECLARE NATIVE CONST KEY_REDO           :Number % redo key
+DECLARE NATIVE CONST KEY_REFERENCE      :Number % ref(erence) key
+DECLARE NATIVE CONST KEY_REFRESH        :Number % refresh key
+DECLARE NATIVE CONST KEY_REPLACE        :Number % replace key
+DECLARE NATIVE CONST KEY_RESTART        :Number % restart key
+DECLARE NATIVE CONST KEY_RESUME         :Number % resume key
+DECLARE NATIVE CONST KEY_SAVE           :Number % save key
+DECLARE NATIVE CONST KEY_SBEG           :Number % shifted beginning key
+DECLARE NATIVE CONST KEY_SCANCEL        :Number % shifted cancel key
+DECLARE NATIVE CONST KEY_SCOMMAND       :Number % shifted command key
+DECLARE NATIVE CONST KEY_SCOPY          :Number % shifted copy key
+DECLARE NATIVE CONST KEY_SCREATE        :Number % shifted create key
+DECLARE NATIVE CONST KEY_SDC            :Number % shifted delete char key
+DECLARE NATIVE CONST KEY_SDL            :Number % shifted delete line key
+DECLARE NATIVE CONST KEY_SELECT         :Number % select key
+DECLARE NATIVE CONST KEY_SEND           :Number % shifted end key
+DECLARE NATIVE CONST KEY_SEOL           :Number % shifted clear line key
+DECLARE NATIVE CONST KEY_SEXIT          :Number % shifted exit key
+DECLARE NATIVE CONST KEY_SFIND          :Number % shifted find key
+DECLARE NATIVE CONST KEY_SHOME          :Number % shifted home key
+DECLARE NATIVE CONST KEY_SIC            :Number % shifted input key
+
+DECLARE NATIVE CONST KEY_SLEFT          :Number % shifted left arrow key
+DECLARE NATIVE CONST KEY_SMESSAGE       :Number % shifted message key
+DECLARE NATIVE CONST KEY_SMOVE          :Number % shifted move key
+DECLARE NATIVE CONST KEY_SNEXT          :Number % shifted next key
+DECLARE NATIVE CONST KEY_SOPTIONS       :Number % shifted options key
+DECLARE NATIVE CONST KEY_SPREVIOUS      :Number % shifted prev key
+DECLARE NATIVE CONST KEY_SPRINT         :Number % shifted print key
+DECLARE NATIVE CONST KEY_SREDO          :Number % shifted redo key
+DECLARE NATIVE CONST KEY_SREPLACE       :Number % shifted replace key
+DECLARE NATIVE CONST KEY_SRIGHT         :Number % shifted right arrow
+DECLARE NATIVE CONST KEY_SRSUME         :Number % shifted resume key
+DECLARE NATIVE CONST KEY_SSAVE          :Number % shifted save key
+DECLARE NATIVE CONST KEY_SSUSPEND       :Number % shifted suspend key
+DECLARE NATIVE CONST KEY_SUNDO          :Number % shifted undo key
+DECLARE NATIVE CONST KEY_SUSPEND        :Number % suspend key
+DECLARE NATIVE CONST KEY_UNDO           :Number % undo key
+
+% PDCurses-specific key definitions -- PC only
```

Comment:
What do you intend to do with the key definitions that appear in PDcurses but not in ncurses? Currently those are causing a build failure.